cuegui/cuegui/HostMonitor.py

```diff
@@ -30,6 +30,7 @@
 
 
 log = cuegui.Logger.getLogger(__file__)
+settings = QtGui.qApp.settings
```

Comment:
nit: My preference would be to omit the local variable and just reference `QtGui.qApp.settings` directly. A bit more typing but it eliminates any ambiguity about what `settings` is.

Reply:
Updated accordingly.